doc/manpages/ocne-config.yaml.md

```diff
@@ -112,6 +112,16 @@ clusterDefinition: mycluster.yaml
 clusterDefinitionInline: |
   somekey: someval
   someotherkey: someotherval
+  
+# This field specifies the password set for the ocne user.
```

Comment:
Use "Oracle Cloud Native Environment" instead of "ocne".  There are a couple of instances.

Reply:
In this case I use ocne literally in reference to the "ocne" user on OCK. I can make that more clear by using quotes. In other places I will be sure to switch out the acronym